submissions/DejoneaRoberts/DOM/productCost.js

```diff
@@ -0,0 +1,64 @@
+let quantity = document.getElementById('quantity'),
+    price = document.getElementById('price'),
+    tempShipping = 0,
+    shipping = document.getElementById('shipping')
+    totalCost = document.getElementById('total-cost'),
+    button = document.getElementById('buy-btn'),
+    perUnit = price.innerHTML,
+    productPrice = perUnit * quantity.value,
+    numRegex = /^[1-9]+[0-9]*$/;
+
+function quantityError() {
+    if (!quantity.value.match(numRegex))  {
+        button.disabled = true;
+        quantity.nextElementSibling.nextElementSibling.innerHTML = "Please enter a valid quantity";
+    }else {
+        button.disabled = false;
+        quantity.nextElementSibling.innerHTML = "";
+    }
+}
+
+function priceCalc() {
+        if (quantity.value > 0) {
+        productPrice = quantity.value * perUnit;

```

Comment:
This same calculation is already been done in line 8. This is more of a repetition